packages/service/tests/acceptance/test_measurement_service.py

```diff
@@ -154,6 +160,8 @@ def test___measurement_service_v1___measure_with_large_array___returns_output(
     protobuf_enum_in = ProtobufColor.WHITE
     string_array_in = ["", "TestString1", "#$%!@<*(&^~`"]
 
+    metadata = stub_v1.GetMetadata(v1_measurement_service_pb2.GetMetadataRequest())
```

Comment:
These tests have arrange/act/assert blocks separated by one blank line. Remove the blank line above metadata so it's part of the arrange block.